src/components/Modals/ReservationConfirmModal/ReservationConfirmModal.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Button, Modal } from "@mui/material";
+import styles from "./ReservationConfirmModal.module.css";
+
+type Props = {
+  openModal: boolean;
+  closeHandler: () => void;
+  confirmBooking: () => void;
+  accData: any;
```

Comment:
Maybe it'd be better if you created the type for accomodation data instead of using `any`. You can look at the swagger docs for reference.


There is also a VSCode plugin for converting json to a typescript type. [Link](https://marketplace.visualstudio.com/items?itemName=quicktype.quicktype)  
Just copy the JSON object in the swagger example and run the command and it will create a type from the JSON and paste it into a file.